metrics/config.go

```diff
@@ -164,18 +176,28 @@ func createMetricsConfig(ops ExporterOptions, logger *zap.SugaredLogger) (*metri
 		// Use Prometheus if DEFAULT_METRICS_BACKEND does not exist or is empty
 		backend = string(Prometheus)
 	}
-	// Override backend if it is setting in config map.
+	// Override backend if it is set in the config map.
 	if backendFromConfig, ok := m[BackendDestinationKey]; ok {
 		backend = backendFromConfig
 	}
 	lb := metricsBackend(strings.ToLower(backend))
 	switch lb {
-	case Stackdriver, Prometheus:
+	case Stackdriver, Prometheus, OpenCensus:
 		mc.backendDestination = lb
 	default:
 		return nil, fmt.Errorf("unsupported metrics backend value %q", backend)
 	}
 
+	if mc.backendDestination == OpenCensus {
+		mc.collectorAddress = ops.ConfigMap[collectorAddressKey]
+		if isSecure := ops.ConfigMap[collectorSecureKey]; isSecure != "" {
```

Comment:
Does strconv.ParseBool consider "" to be valid? If not, maybe just:
if isSecure, ok := ops.ConfigMap[collectorSecureKey]; ok {
// let empty string return invalid value error
}

Reply:
It does not. I can add a test for that if you want.

https://golang.org/pkg/strconv/#ParseBool
> It accepts 1, t, T, TRUE, true, True, 0, f, F, FALSE, false, False. Any other value returns an error.

Reply:
Oh, I don't think a test case is necessary to validate ParseBool behavior. I was just thinking to also error as invalid if someone uses empty string i.e. "metrics.collector-require-tls="
		if isSecure, ok := ops.ConfigMap[CollectorSecureKey]; ok {
			var err error
			if mc.requireSecure, err = strconv.ParseBool(isSecure); err != nil {
				return nil, fmt.Errorf("invalid %s value %q", CollectorSecureKey, isSecure)
			}
		}